app/views/components/_govspeak-editor.html.erb

```diff
@@ -70,11 +69,12 @@
       data: textarea_data_attributes,
       margin_bottom: 0,
       describedby: hint_id,
-      right_to_left: right_to_left
+      right_to_left: right_to_left,
+      right_to_left_help: false
     } %>
   </div>
 
-  <%= tag.div class: "app-c-govspeak-editor__preview", 'aria-live': "polite", id: preview_id, dir: dir do %>
+  <%= tag.div class: "app-c-govspeak-editor__preview js-locale-switcher-custom", 'aria-live': "polite", id: preview_id, dir: dir do %>
     <p class="govuk-body">Generating preview, please wait.</p>
```

Comment:
Should this `<p>` have an explicit `dir="ltr"` for the sake of completeness?

```suggestion
    <p class="govuk-body" dir="ltr">Generating preview, please wait.</p>
```

So that this part of the UI still displays correctly, even if the overall document is in an RTL language.

Reply:
Yes, I've added that - good spot.